src/main/java/io/nats/client/api/ConsumerConfiguration.java

```diff
@@ -981,11 +981,11 @@ public String toString() {
      * a default value.
      */
     public enum CcChangeHelper {
-        START_SEQ(1, -1),
+        START_SEQ(1, 0),
         MAX_DELIVER(1, -1),
-        RATE_LIMIT(1, -1),
-        MAX_ACK_PENDING(0, 0),
-        MAX_PULL_WAITING(0, 0),
+        RATE_LIMIT(1, 0),
+        MAX_ACK_PENDING(0, -1),
+        MAX_PULL_WAITING(0, -1),
```

Comment:
start seq and rate limit are unsigned, so 1 didn't make sense. max ack pending and max pull waiting are reverted back to they way they were in 2.14.0 (this is 2.14.1)